src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VcfLineMapper.java

```diff
@@ -33,12 +36,20 @@
     private final VariantVcfFactory factory;
 
     public VcfLineMapper(VariantSource source) {
+        if (!VariantSource.Aggregation.NONE.equals(source.getAggregation())) {
```

Comment:
[GATK also use the "genotyped VCF" term](https://software.broadinstitute.org/gatk/gatkdocs/org_broadinstitute_gatk_tools_walkers_variantutils_GenotypeGVCFs.php), so at least it should be understable.